src/app/fdctl/run/run.c

```diff
@@ -649,7 +649,7 @@ check_configure( config_t * const config ) {
                  "to create the mounts correctly. This must be done after every system restart before running "
                  "Firedancer.", check.message ));
 
-  check = ethtool.check( config );
+  check = ethtool_channels.check( config );
   if( FD_UNLIKELY( check.result!=CONFIGURE_OK ) )
     FD_LOG_ERR(( "Network %s. You can run `fdctl configure init ethtool` to set the number of channels on the "
```

Comment:
`fdctl configure init ethtool-channels`

Reply:
Can you also find / replace in the codebase to make sure there aren't other cases like this

Reply:
will do